bifrost/app/blog/blogs/test-your-llm-prompts/src.mdx

```diff
@@ -0,0 +1,286 @@
+
+|                | **Data collection (Datasets)** | **Prompt Management** | **Playground** | **Evaluations** | **Experiments** | **Open-source** |
+| -------------- | ------------------------------ | --------------------- | -------------- | --------------- | --------------- | --------------- |
+| **Helicone**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Langfuse**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Arize AI**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Braintrust** | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ❌              |
+| **LangSmith**  | ✔️                             | ✔️                    | ✔️             | ✔️              | ❌              | ❌              |
+
+## Components to improve LLM prompts
+
+### Data collection
+
+You need high-quality input variables to create high-quality prompts. A common practice is to use “**<span style={{color: '#0ea5e9'}}>golden datasets</span>**” - data that’s meticulously cleaned and labeled - to tune your prompt for problems that are easily reproducible.
+
+Complementing this approach, teams like QA Wolf also use **<span style={{color: '#0ea5e9'}}>randomly sampled production data</span>**, which represents actual user interactions and offers a more accurate representation of real-world use cases. QA Wolf uses Helicone as their observability tool to log every LLM request and export comprehensive datasets that can be used to further fine-tune their LLMs.
+
+### Prompt management
+
+Prompt registries are specialized tools to help engineers **<span style={{color: '#0ea5e9'}}>version, manage or improve prompts</span>**, and roll back to a previous version easily.
+
+### Prompt playgrounds
+
+Playgrounds are **<span style={{color: '#0ea5e9'}}>interactive environments</span>** that helps engineers iteratively test and refine their prompts. Notably, playgrounds allows you to view detailed prompt/response pairs, create and iterate on prompt templates, experiment with model configurations and test prompts in real-time.
+
+### Evaluators & scores
+
+Feedback is essential for continuous improvement. The best tools close the loop with **<span style={{color: '#0ea5e9'}}>quantitative scoring</span>** and **<span style={{color: '#0ea5e9'}}>scalable quality assessment</span>** of LLM outputs. They make it easy to compare different prompt variations and help engineers justify prompt changes with data before pushing to production
```

Comment:
What does this mean? The best tools close the loop with **<span style={{color: '#0ea5e9'}}>quantitative scoring</span>** and **<span style={{color: '#0ea5e9'}}>scalable quality assessment</span>** of LLM outputs.

Reply:
close the loop = last step of the life cycle, but i agree it can be written more clearly. 

How is "The best tools analyze LLM outputs through qualitative and quantitative metrics."